integrationExamples/testBidder/testBidderBannerExample.html

```diff
@@ -0,0 +1,79 @@
+<html>
+    <head>
+        <title>Prebid Test Bidder Example</title>
+        <script src="../../build/dev/prebid.js" async></script>
+        <script>
+            var pbjs = pbjs || {};
+            pbjs.que = pbjs.que || [];
+
+            const adUnitCode = 'adUnit-0000';
+
+            const adUnits = [{
+                mediaTypes: {
+                    banner: {
+                        sizes: [600, 500]
+                    }
+                },
+                code: adUnitCode,
+                bids: [
+                    {bidder: 'testBidder', params: {}}
+                ]
+            }]
+
+            pbjs.que.push(function () {
+
+                /**
+                 * BID RESPONSE SIMULATION SECTION START
+                 *
+                 * This section handles simulating a bidder
+                 * that will always respond with bid responses.
+                 *
+                 * This part should not be present in production.
+                 */
+                pbjs.registerBidAdapter(null, 'testBidder', {
+                    supportedMediaTypes: ['banner', 'video', 'native'],
+                    isBidRequestValid: () => true
+                });
+
+                pbjs.setConfig({
+                    debugging: {
+                        enabled: true,
+                        intercept: [
+                            {
+                                when: {
+                                    bidder: 'testBidder',
+                                },
+                                then: {
+                                    creativeId: 'testCreativeId',
```

Comment:
is this just to have something here? I think it should work with an empty object.